go-controller/pkg/cni/cniserver.go

```diff
@@ -179,7 +179,8 @@ func cniRequestToPodRequest(cr *Request, podLister corev1listers.PodLister, kcli
 
 	req.CNIConf = conf
 	req.timestamp = time.Now()
-	req.ctx, req.cancel = context.WithTimeout(context.Background(), time.Minute)
+	// Match the Kubelet default CRI operation timeout of 2m
+	req.ctx, req.cancel = context.WithTimeout(context.Background(), 2*time.Minute)
```

Comment:
@tssurya kubelet has a bound of 4 minutes for StartPodSandbox actually. I don't really know why CRIO uses 5, but unless you override the default kubelet CRI timeout, it'll be shorter than CRIO's and the sandbox add will fail anyway.

I picked 2 minutes because we saw some pods taking 1m4s in the 500 node bare-metal tests, and I can't think of a good reason to make the timeout significantly less than the kubelet/CRI timeout, otherwise we just get into kubelet backoff behavior which makes things worse.